include/deal.II/fe/fe_base.h

```diff
@@ -242,6 +242,13 @@ class FiniteElementData
    */
   const unsigned int dofs_per_hex;
 
+  /**
+   * Total number of global degrees of freedom on a cell. This is the
+   * number of degrees of freedom not within this cell.
```

Comment:
This is a new kind of DoF for an FE that doesn't fit into the other categories? Do you have an example? Why is special about a DoF that is not per_hex?  (put this into the comment here). 
Also add a sentence like "This is the number of global DoFs, not including the DoFs on the hex, quad, line, vertex (above)."

Finally, I am not sure that "global" is a great name (but I also don't have a better idea except "other").

Reply:
@tjhei , see #7757 for a discussion on this

Reply:
Would non_local not be a better name than global?

Reply:
> Would non_local not be a better name than global?

yes, it would https://github.com/dealii/dealii/issues/7757#issuecomment-467716665

and it looks like we all agree on that point ;-)